### PR TITLE
Support  .webp image format  #6716

### PR DESCRIPTION
**Pull request by Aijeyomah**
**What does this PR do?**
- resolve  include `.webp` image format in the list of supported file type images.

** Loom  Screen record of How it works **
https://www.loom.com/share/46fb3044e72947b09d16f24529ce2cbc

### CODE REVIEW

File: test/mocks/images/city.webp
Comment (Betree) [inline test/mocks/images/city.webp:1]: Looks like this image is [copywrited](https://www.alamy.com/stock-photo-cityscape-of-london-and-big-ben-172776427.html). Let's use something else, any homemade image with a few colors will do the job.

File: test/server/routes/images.test.js [hunk 1]
```diff
@@ -48,6 +48,24 @@ describe('server/routes/images', () => {
...
+    });
+
+    const res = await request(expressApp)
+      .post(`/images/?api_key=${application.api_key}`)
+      .attach('file', 'test/mocks/images/city.webp')
+      .field('kind', 'ACCOUNT_LONG_DESCRIPTION')
+      .set('Authorization', `Bearer ${user.jwt()}`);
+
+    expect(res.status).to.eq(200);
+    expect(res.body.url).to.contain('.png');
```
Comment (Betree) [inline test/server/routes/images.test.js:63]:
```suggestion
    expect(res.body.url).to.contain('.webp');
```

**Comment by Aijeyomah** [comment]
@Betree  Here's the link to FE the https://github.com/opencollective/opencollective-frontend/pull/8903

**Comment by Betree** [comment]
Rebased

**Review by Betree — APPROVED** [review]
Tested locally, works perfectly. Great work @Aijeyomah!

**Pull request merged**